contacts.routing.yml

```diff
@@ -28,3 +28,10 @@ contacts.add_org_form:
     _permission: 'add contacts'
   options:
     _admin_route: TRUE
+
+contacts.ajax_update_blocks:
+  path: '/admin/contacts-ajax/update-layout'
```

Comment:
Would it be more approapriate to use `/admin/contacts/ajax/...`?

Reply:
sure